clang/lib/DPCT/AnalysisInfo.cpp

```diff
@@ -2825,6 +2825,7 @@ std::string CallFunctionExpr::getTemplateArguments(bool &IsNeedWarning,
     if ((TA.isNull() || !TA.isWritten()) && WrittenArgsOnly)
       continue;
     std::string Str = TA.getString();
+
```

Comment:
there is no logic change here, to keep small footprint, so please revert this change.